staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -290,27 +314,58 @@ func subjectAccessReviewInterfaceFromKubeconfig(kubeConfigFile string, version s
 }
 
 type subjectAccessReviewV1Client struct {
-	w *webhook.GenericWebhook
+	client rest.Interface
 }
 
-func (t *subjectAccessReviewV1Client) Create(ctx context.Context, subjectAccessReview *authorizationv1.SubjectAccessReview, _ metav1.CreateOptions) (*authorizationv1.SubjectAccessReview, error) {
+func (t *subjectAccessReviewV1Client) Create(ctx context.Context, subjectAccessReview *authorizationv1.SubjectAccessReview, opts metav1.CreateOptions) (result *authorizationv1.SubjectAccessReview, statusCode int, err error) {
```

Comment:
isn't it possible to derive the status code from the error returned?

Reply:
It is. Huh, didn't the previous versions disallowed it?

Reply:
please ignore my previous comment. 

We should not try to derive the status code from the error. 
The `client-go` returns the `Result` struct which contains `err` and `status code` as distinct fields this is because there is no guarantee that the server responds with `metav1.Status`